Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -374,12 +374,18 @@ open class TextStorage: NSTextStorage {
             attachment.delegate = self
         }
         
-        textStore.enumerateAttachmentsOfType(CommentAttachment.self) { [weak self] (attachment, _, _) in
-            attachment.delegate = self
-        }
-        
-        textStore.enumerateAttachmentsOfType(HTMLAttachment.self) { [weak self] (attachment, _, _) in
+        enumerateRenderableAttachments(in: textStore, block: { [weak self] (attachment, _, _) in
             attachment.delegate = self
+        })
+                
+    }
+
+    private func enumerateRenderableAttachments(in text: NSAttributedString, range: NSRange? = nil, block: ((RenderableAttachment, NSRange, UnsafeMutablePointer<ObjCBool>) -> Void)) {
```

Comment:
This method is currently in the`HTML Interaction` code section, which doesn't sound right.

Can we move this to the `Attachments` code section above?

(ie: Pragma marks)